common/VotingUtils.js

```diff
@@ -116,20 +92,20 @@ const batchCommitVotes = async (newCommitments, votingContract, account) => {
     const maxBatchSize = newCommitments.slice(k, Math.min(k + BATCH_MAX_COMMITS, newCommitments.length));
 
     // Always call `batchCommit`, even if there's only one commitment. Difference in gas cost is negligible.
-    const { receipt } = await votingContract.batchCommit(
-      maxBatchSize.map(commitment => {
-        // This filters out the parts of the commitment that we don't need to send to solidity.
-        // Note: this isn't strictly necessary since web3 will only encode variables that share names with properties in
-        // the solidity struct.
-        const { price, salt, ...rest } = commitment;
-        return rest;
-      }),
-      { from: account }
-    );
-
+    const { transactionHash } = await votingContract.methods
+      .batchCommit(
+        maxBatchSize.map(commitment => {
+          // This filters out the parts of the commitment that we don't need to send to solidity.
+          // Note: this isn't strictly necessary since web3 will only encode variables that share names with properties in
+          // the solidity struct.
+          const { price, salt, ...rest } = commitment;
+          return rest;
+        })
+      )
+      .send({ from: account, gas: gasToSendTransactions });
```

Comment:
@mrice32 @chrismaree @adrianmcli having to explicitly specify `gas` is a new web3.contract thing we have to deal with

Reply:
Wait, I thought we didn't need to do these new constructions now that we're running from `core`? Artifacts should _just work_ right?

Reply:
Well I wanted to decouple the CLI (and eventually other clients) away from the truffle framework, so that we can start the program without using truffle exec. This will prevent us from having to start the program from within core. 

If this isn’t a priority item then I can just stop working on it now. But this is a preview of what the clients would begin to look like without being tied to Truffle.